kroxylicious/example-proxy-config.yml

```diff
@@ -18,6 +18,6 @@ clusters:
 filters:
 - type: ApiVersions
 - type: BrokerAddress
-- type: ProduceRequestTransformation
-  config:
-    transformation: io.kroxylicious.proxy.internal.filter.ProduceRequestTransformationFilter$UpperCasing
+#- type: ProduceRequestTransformation
```

Comment:
did you mean to leave this commented @franvila?

Reply:
Yes, Tom told me in the chat that this is not relevant to a real user and it can affect to the performance results. I left it commented because it is not an obvious config